lighthouse-core/audits/seo/font-size.js

```diff
@@ -157,12 +157,19 @@ function findStyleRuleSource(baseURL, styleDeclaration, node) {
         source = `${url.href}`;
 
         if (range) {
-          // `stylesheet` can be either an external file (stylesheet.startLine will always be 0),
```

Comment:
does this mean this PR actually changes any behavior for the `line`? or was the previous comment incorrect and sometimes `stylesheet.startLine` is not 0?

Reply:
It's only zero if not from a style block. I can add it back - although, the value is only used now if the stylesheet is inline, so it being zero in the other case is not important anymore.

Reply:
right, so the new version and the old version have the exact same inputs/outputs just making sure 👍